### ISSUE TITLE
`make clean` stopped by .DS_Store file

### ISSUE THREAD

**Issue by EricFromCanada** [sphinx-doc/sphinx#2781]
On v1.4.5+, when attempting to rebuild the Sphinx docs on OS X, sphinx.make_mode couldn't handle the auto-generated .DS_Store file.

```
$ make clean
NotADirectoryError: [Errno 20] Not a directory: '_build/.DS_Store'
make: *** [clean] Error 1
```

**Comment by tk0miya** [comment]
Fortunately, now we have PR #2777. It will resolve this.

**Comment by tk0miya** [comment]
I merged #2782 into stable branch.
Thank you @shibukawa!
